src/entity/UserType.ts

```diff
@@ -0,0 +1,17 @@
+import {
+    Entity,
+    Column, 
+    PrimaryGeneratedColumn, 
+    BaseEntity
+} from "typeorm";
+
+@Entity("UserType")
+export class UserType extends BaseEntity {
```

Comment:
I think we could use a constant in the user class to represent the type of user.

Reply:
Solved. We using constants now. I removed the table UserType